src/lein_tools_deps/plugin.clj

```diff
@@ -156,7 +155,16 @@
        (resolve-deps (:root project))
        (merge project)))
 
-(defn middleware
+(defn ^:deprecated middleware
+  "Inject relevant keys from deps.edn files into the leiningen project map
+  while honoring other user-supplied config."
+  [_]
+  (lein/warn "Implicit middleware usage is deprecated."
+              " Please use explicit middleware instead."
+              " See https://github.com/RickMoynihan/lein-tools-deps/issues/38")
```

Comment:
Instead of linking to the issue here it might be better to just say:

> Implicit middleware usage has been removed from lein-tools-deps.  Please use an explicit middleware call instead by including the following line in your leiningen project:
>
> :middleware [lein-tools-deps.plugins/resolve-dependencies-with-deps-edn]